pkg/internal/wordpress/pod_template.go

```diff
@@ -89,6 +98,81 @@ func (wp *Wordpress) hasExternalMedia() bool {
 		(wp.Spec.MediaVolumeSpec.S3VolumeSource != nil || wp.Spec.MediaVolumeSpec.GCSVolumeSource != nil)
 }
 
+func (wp *Wordpress) installCredentials() *wpBootstrapConfig {
```

Comment:
Use the `.Spec.Bootstrap.Env` and `.Spec.Bootstrap.EnvFrom` as is and inject them into the init container.